src/utils/companyNumberSanitizer.ts

```diff
@@ -0,0 +1,37 @@
+import { provide } from 'inversify-binding-decorators'
+
+@provide(CompanyNumberSanitizer)
+export default class CompanyNumberSanitizer {
+  private readonly COMPANY_NUMBER_SIZE: number = 8
+
+  public sanitizeCompany(companyNumber: string): string {
+    if (!companyNumber) {
+      throw new Error('Company number is required')
+    }
```

Comment:
Does that mean that sanitizeCompany should accept an Optional<string> instead?

Reply:
How does the UI react to an empty company number being provided? If this error is thrown then the validation schema is never executed. I'm concerned that this will show the global error page rather than a GDS inline error.

Reply:
Yep, removed.